tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetWebApi2Tests.cs

```diff
@@ -61,6 +61,51 @@ public AspNetWebApi2TestsCallTargetIntegratedWithRouteTemplateExpansion(IisFixtu
             : base(iisFixture, output, classicMode: false, enableRouteTemplateResourceNames: true, enableRouteTemplateExpansion: true)
         {
         }
+
+        [Collection("IisTests")]
+        public class AspNetWebApi2TestsVirtualAppIntegratedWithFeatureFlag : AspNetWebApi2Tests
+        {
+            public AspNetWebApi2TestsVirtualAppIntegratedWithFeatureFlag(IisFixture iisFixture, ITestOutputHelper output)
+                : base(iisFixture, output, virtualApp: true, classicMode: false, enableRouteTemplateResourceNames: true)
+            {
+            }
+        }
+
+        [Collection("IisTests")]
+        public class AspNetWebApi2TestsModuleOnlyClassic : AspNetWebApi2ModuleOnlyTests
+        {
+            public AspNetWebApi2TestsModuleOnlyClassic(IisFixture iisFixture, ITestOutputHelper output)
+                : base(iisFixture, output, virtualApp: false, classicMode: true, enableRouteTemplateResourceNames: false)
+            {
+            }
+        }
+
+        [Collection("IisTests")]
+        public class AspNetWebApi2TestsModuleOnlyIntegrated : AspNetWebApi2ModuleOnlyTests
+        {
+            public AspNetWebApi2TestsModuleOnlyIntegrated(IisFixture iisFixture, ITestOutputHelper output)
+                : base(iisFixture, output, virtualApp: false, classicMode: false, enableRouteTemplateResourceNames: false)
+            {
+            }
+        }
+
+        [Collection("IisTests")]
+        public class AspNetWebApi2TestsModuleOnlyVirtualAppIntegrated : AspNetWebApi2ModuleOnlyTests
+        {
+            public AspNetWebApi2TestsModuleOnlyVirtualAppIntegrated(IisFixture iisFixture, ITestOutputHelper output)
+                : base(iisFixture, output, virtualApp: true, classicMode: false, enableRouteTemplateResourceNames: false)
+            {
+            }
+        }
+
+        [Collection("IisTests")]
+        public class AspNetWebApi2TestsModuleOnlyVirtualAppIntegratedWithFeatureFlag : AspNetWebApi2ModuleOnlyTests
+        {
+            public AspNetWebApi2TestsModuleOnlyVirtualAppIntegratedWithFeatureFlag(IisFixture iisFixture, ITestOutputHelper output)
+                : base(iisFixture, output, virtualApp: true, classicMode: false, enableRouteTemplateResourceNames: false)
```

Comment:
```suggestion
                : base(iisFixture, output, virtualApp: true, classicMode: false, enableRouteTemplateResourceNames: true)
```

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2851/commits/5ec0eea5483d97f3a5043add3d4df0978eec92d5